protos/manual_control/manual_control.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package mavsdk.rpc.manual_control;
+
+option java_package = "io.mavsdk.manual_control";
+option java_outer_classname = "Manual_ControlProto";
+
+service ManualControlService
+{
+
+    
+    /*
+     * Send command to enable Joystick mode 
+     *
+     * Enabling Joystic mode causes to change into Position Control mode
+     * drone still needs to be armed
+     * Needs to check if the drone is not currently flying or in mission if so drone needs to be paused first 
+     */
+    
+    rpc SetPositionControl (PositionControlRequest)  returns(PositionControlResponse) {}
+    
+    
+    rpc SetAltitudeControl (AltitudeControlRequest)  returns(AltitudeControlResponse) {}
+    
+      /*
+     * Send command to command to the drone to move forwards or backwards 
+     */
+    rpc Axismotion (MotionRequest) returns (MotionResponse) {}
```

Comment:
Axismotion is a term I don't know. What about `ManualControlInput` or `ManualSetpoint`?